src/desktop/apps/articles/routes.ts

```diff
@@ -20,12 +20,16 @@ let positronql = _positronql
 let topParselyArticles = _topParselyArticles
 let stitch = _stitch
 
-export const articles = (_req, res, next) => {
-  const query = { query: magazineQuery }
+export const articles = (req, res, next) => {
```

Comment:
Would it be possible to add some tests for this router? In particular, it would be good to test that it can handle inputs like `page=foo`, `page=0`, `page=` and no page param at all. (Recalling these were what we verified manually).

Reply:
Yeah definitely, I'll add a test for that.

Reply:
Good call on writing a test here, we missed negative page values on the first pass!